core/shared/utils/bh_common.c

```diff
@@ -167,15 +167,39 @@ wa_strdup(const char *s)
 }
 
 #if WASM_ENABLE_WAMR_COMPILER != 0 || WASM_ENABLE_JIT != 0
+/* need to make sure that The `argv[]` must be terminated by a NULL pointer. */
 int
-bh_system(const char *cmd)
+bh_system(const char *pathname, char *const argv[], int argc)
```

Comment:
i feel the name bh_system is confusing if it doesn't work like system().

Reply:
Got you points. I will rename it to bh_execve().